tests/utest/test_default_config.py

```diff
@@ -232,3 +232,15 @@ def test_override_default_config(self, path_to_test_data):
                 config = Config(from_cli=True)
             assert str(config.config_from) == str(other_config)
             assert config.include == {"0810"}
+
+
+def test_nested_argument_files(path_to_test_data):
```

Comment:
I think this is the part of another PR #774 and should be removed from here.

Reply:
Rebased after merging #774